### ISSUE TITLE
Peer dependencies causing problem - multiple versions of ui-core

### ISSUE THREAD

**Issue by elfoxus** [goetzrobin/spartan#263]
### Please provide the environment you discovered this bug in.

Windows 11, npm 10.2.4, node 21.6.0, angular 17 cli

### Which area/package is the issue in?

Don't know / other

### Description

Hi. After installing spartan components following tutorial I cannot run npm install. I've noticed that some components have different versioning than the others, i.e. `0.0.1-alpha.344` and `0.0.1-alpha.343`. It might be causing problems.
I dont think that using angular cli made any changes, cause same thing happened with nx.

What might be the solution to this topic? Not using components with old version so no mismatch is happening? Thanks for help.

```bash
 code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: frontend@0.0.0
npm ERR! Found: @spartan-ng/ui-core@0.0.1-alpha.344
npm ERR! node_modules/@spartan-ng/ui-core
npm ERR!   @spartan-ng/ui-core@"0.0.1-alpha.344" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @spartan-ng/ui-core@"0.0.1-alpha.343" from @spartan-ng/ui-checkbox-brain@0.0.1-alpha.344
npm ERR! node_modules/@spartan-ng/ui-checkbox-brain
npm ERR!   @spartan-ng/ui-checkbox-brain@"0.0.1-alpha.344" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by DevWedeloper** [comment]
This happened to me too when installing @spartan-ng/ui-dialog-brain (with other spartan brains already existing), currently relying on running `npm install --legacy-peer-deps` until a fix occurs.

**Comment by goetzrobin** [comment]
This issue was addressed as part of releases: cli 0.0.1-alpha.350 & ui 0.0.1-alpha.347.
Let me know if it still occurs
